tests/countries/test_antigua_and_barbuda.py

```diff
@@ -0,0 +1,227 @@
+
+
+class TestAntiguaAndBarbuda(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(AntiguaAndBarbuda, years=range(1955, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(AntiguaAndBarbuda, AG, ATG)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(AntiguaAndBarbuda(years=1954))
+
+    def test_special_holidays(self):
+        self.assertHolidayName("Public Holiday", "1993-08-03")
+        self.assertHolidayName(
+            "State Funeral of the late The Honourable Charlesworth T. Samuel", "2008-02-19"
+        )
+        self.assertHolidayName(
+            "State Funeral of the late The Honourable Sir George Herbert Walter", "2008-03-18"
+        )
+        self.assertHolidayName(
+            "Day after the General Election",
+            "2018-03-22",
+            "2023-01-19",
+        )
+
+    def test_new_years_day(self):
+        name = "New Year's Day"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1955, 2050)))
+        dt = (
+            "1995-01-02",
+            "2006-01-02",
+            "2012-01-02",
+            "2017-01-02",
+            "2023-01-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_good_friday(self):
+        name = "Good Friday"
+        self.assertHolidayName(
+            name,
+            "2021-04-02",
+            "2022-04-15",
+            "2023-04-07",
+            "2024-03-29",
+            "2025-04-18",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_easter_monday(self):
+        name = "Easter Monday"
+        self.assertHolidayName(
+            name,
+            "2021-04-05",
+            "2022-04-18",
+            "2023-04-10",
+            "2024-04-01",
+            "2025-04-21",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_labour_day(self):
+        name = "Labour Day"
+        self.assertHolidayName(
+            name,
+            "2021-05-03",
+            "2022-05-02",
+            "2023-05-01",
+            "2024-05-06",
+            "2025-05-05",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_whit_monday(self):
+        name = "Whit Monday"
+        self.assertHolidayName(
+            name,
+            "2021-05-24",
+            "2022-06-06",
+            "2023-05-29",
+            "2024-05-20",
+            "2025-06-09",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_caribbean_community_day(self):
+        name = "Caribbean Community (Caricom) Day"
+        self.assertHolidayName(
+            name,
+            "1995-07-03",
+            "2000-07-03",
+            "2004-07-05",
+            "2005-07-04",
+        )
+        self.assertNoHoliday(
+            "2010-07-05",
+            "2015-07-06",
+            "2020-07-06",
+        )
```

Comment:
AFAIK it's already covered in `self.assertNoHolidayName(name, range(2006, 2050))`
```suggestion
```